Makefile

```diff
@@ -96,16 +122,23 @@ db-seed-all: py-venv-check
 	python -m ambuda.seed.dictionaries.shabdasagara
 	python -m ambuda.seed.dictionaries.vacaspatyam
 
+# Local run commands
+# ===============================================
+devserver: 
```

Comment:
Let's keep this command and have it run `make mode=dev docker-start`. `devserver` is a more obvious name, and this way the command syntax is clearer for beginners.

Reply:
done